README.md

```diff
@@ -1,26 +1,29 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# World Weather
 
-> Description the project.
+> This is a SPA react-app project that is built using two APIs. And users can select and choose countries and states and get their updated weather info. 
+I have built this project using react-redux and JavaScript.
 
 
 ## Built With 
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5
+- CSS3
+- JavaScript
+- React
+- react-redux
+- Webpack
+- Jest
 
-## Live Demo (if available) 💻
+## Live Demo (Netlify) 💻
 
-[Live Demo Link](https://livedemo.com)
+- [World Weather](https://development--gregarious-pavlova-aaa751.netlify.app/)
 
+- [Live Demo](https://www.loom.com/share/3b83c1bafd504fa09bbfc763cc2f3afc)
```

Comment:
- [ ] Please make sure to talk about a feature or a piece of code you're proud of, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.) requirement

Reply:
Well resolved 👍🏽